ext/opcache/ZendAccelerator.c

```diff
@@ -2287,8 +2290,10 @@ static void accel_activate(void)
 		realpath_cache_clean();
 
 		accel_reset_pcre_cache();
+		ZCG(pcre_reseted) = 0;
```

Comment:
I don't really understand what's going on here. The cache is reset in the line above, but pcre_reseted is set to 0. Why is it necessary to reset the cache again?

Reply:
Ah, I think I got it. We already finished this restart, so this sets pcre_reseted=0 for the next one.

Reply:
Right. We reseted PCRE cache and set ZCG(pcre_reseted) = 1, to avoid multiple resets during restart pending, then reseted once again to repopulated cache with interned keys, and set ZCG(pcre_reseted) = 0 to be prepared for next restart.